src/server.ts

```diff
@@ -0,0 +1,71 @@
+import * as grpc from 'grpc';
+import * as protoLoader from '@grpc/proto-loader';
+import * as apm from 'elastic-apm-node';
+import { GrpcHealthCheck, HealthCheckResponse, HealthService } from 'grpc-ts-health-check';
+import { config } from './config';
+import { log, Severity } from './utils/logger';
+import { wrapper } from './utils/wrapper';
+import { DropboxMethods } from './dropbox/dropbox.methods';
+
+apm.start({
+    serviceName: config.serviceName,
+    secretToken: config.apm.secretToken,
+    verifyServerCert: config.apm.verifyServerCert,
+    serverUrl: config.apm.apmURL,
+});
```

Comment:
The comment here is just for your own knowledge:
`elastic-apm-node` agent instruments libraries during start up. So that in the common express-mongoose-axios NodeJs app all you need to do for a premium APM experience is just `apm.start`! 😮 .
**BUT** in order to work correctly, apm must be the **first** module that you load-up in your app.
But you're using GRPC so it will probably isn't relevant for you 😝